build.Containerfile

```diff
@@ -0,0 +1,39 @@
+#
+# Reproducible reference build
+#
+# Usage:
+#
+# docker build --file build.Containerfile --output <outputdir> .
+# or
+# podman build --file build.Containerfile --output <outputdir> .
+#
+# The build artifacts are written to the specified output directory.
+#
+
+FROM debian:bullseye-slim AS build-stage
+
+# prepare debian environment
+ENV DEBIAN_FRONTEND noninteractive
+RUN /usr/bin/apt-get update && \
+    /usr/bin/apt-get --yes install openjdk-11-jdk-headless gradle && \
```

Comment:
I'm also hoping (though it is not critical) that we can use bookworm and JDK 17 here.

Reply:
Atm I wanted to resemble the status quo. The switch from JDK 11 to 17 is orthogonal.